src/basic/pager.c

```diff
@@ -41,10 +43,45 @@ _noreturn_ static void pager_fallback(void) {
         _exit(EXIT_SUCCESS);
 }
 
+static inline int no_quit_on_interrupt(int exe_name_fd, const char *less_opts) {
```

Comment:
why the `inline`? I mean, in header files "static inline" makes a lot of sense, but in .c files I am pretty sure the compiler can figure this out perfectly on its own

Reply:
Ah, sorry, this was originally structured differently. I'll rework it.